src/components/Inputs/SearchBox/SearchBox.tsx

```diff
@@ -49,7 +50,13 @@ export const SearchBox: FC<SearchBoxProps> = React.forwardRef(
         ref: Ref<HTMLInputElement>
     ) => {
         return (
-            <form role="search">
+            <form
+                role="search"
+                onSubmit={(_event) => {
+                    _event.preventDefault();
+                    onSubmit?.(_event);
```

Comment:
lets not prevent default here, let the user decide if they want to
```suggestion
                onSubmit={(e) => onSubmit?.(e)}
```

Reply:
@ychhabra-eightfold  done.